__tests__/file-system.test.js

```diff
@@ -0,0 +1,28 @@
+// after all delete newly created directories and files
+const fs = require('fs').promises;
+
+const {
+  mkdirp,
+  deleteFile
+} = require('../lib/file-system.js');
+
+describe('file system', () => {
+    
+  it('makes a directory', () => {
+    mkdirp('./my-directory');
+    fs.readdir('./my-directory', (err, files) => {
```

Comment:
this needs to go into a `.then(() =>)`. you want to wait to read the directory until after the mkdirp finishes